### DISCUSSION TITLE
如何实现如下这种图？

### DISCUSSION THREAD

**Discussion by usekevinwu** [antvis/G2#5926] [Ideas]
基础是由圆形进行分割，先将圆形分割不同模块，然后再将不同再切割，有点类似矩阵树图的功能

![153e934da5128157caf9ab459291e07](https://github.com/antvis/G2/assets/37398179/8613fb1a-0357-46cd-9584-af5a03052b1a)

**Comment by 003d** [comment]
请问原图出处，这个算法挺有意思的

**Reply by usekevinwu** [reply]
无意中看到的；
网上搜到很多，一开始觉得是不是用PPT做的
![image](https://github.com/antvis/G2/assets/37398179/60bdaed1-5d23-49a5-b31a-908a4c6b8fb3)

**Reply by hustcc** [reply]
https://g2-v4.antv.vision/zh/examples/relation/relation#voronoi-polar

v4 版本提供了这个 DEMO，在 v5 中还没有提供，可以考虑试试看，主要还是布局函数实现补布局。

![image](https://github.com/antvis/G2/assets/7856674/6d72050b-5026-43e7-84d6-a07cce085e36)

**Reply by usekevinwu** [reply]
好的，谢谢大佬！
